src/__experimental__/components/fieldset/fieldset-examples.stories.mdx

```diff
@@ -55,7 +55,7 @@ Address Fieldset for Canada.
         label='ZIP Code'
         labelInline
         labelAlign='right'
-        styleOverride={ { input: { width: '120px', flex: 'none' } } }
+        inputWidth={10}
```

Comment:
10*8px = 80px
What's the story here?

Reply:
The old input width included the label. This is the actual input width.